vm-validator/src/unit_tests/vm_validator_test.rs

```diff
@@ -384,16 +384,29 @@ fn test_validate_invalid_arguments() {
 fn test_validate_non_genesis_write_set() {
     let vm_validator = TestValidator::new();
 
+    // Confirm that a correct transaction is validated successfully.
     let address = account_config::libra_root_address();
     let transaction = transaction_test_helpers::get_write_set_txn(
         address,
-        2,
```

Comment:
Should we check for the txn sequence number too new case as well here?

Reply:
There's already a test for that in e2e-testsuite/src/tests/write_set.rs. Also, with this change, "too new" sequence numbers are now OK for validation of WriteSet transactions (just as they are for other kinds of transactions), so there's nothing special to be tested for that. That said, if you think it's important to test here, it wouldn't hurt anything (except time to run the tests) to add another test for that

Reply:
Ah that makes sense! Was just wondering why the test was removed. This sounds good to me then!